src/SNaQ.jl

```diff
@@ -49,6 +51,7 @@ module SNaQ
         readtrees2CF,
         readtableCF,
         readtableCF!,
+        readPhylip2CF,
```

Comment:
To adhere to julian naming convention, you should replace `readPhylip2CF` by `readphylip2CF` perhaps.